src/config.rs

```diff
@@ -390,6 +393,10 @@ impl General {
     pub fn default_prometheus_exporter_port() -> i16 {
         9930
     }
+
+    pub fn default_server_round_robin() -> bool {
```

Comment:
This brings us back to original PgCat behavior before the bb8 queuing strategy change, is that correct?

Reply:
That's correct, using a queue instead of stack as the default. Once we figure out if this is causing connection thrashing we can switch the default and match pgbouncer